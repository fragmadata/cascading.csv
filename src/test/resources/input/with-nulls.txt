id,first name,last name
1,Jordan,Halterman
2,Tom,\N
3,Harrison,Cavallero
4,\N,Ahmed
5,Tristan,Reid
6,Mark,McKelvy
