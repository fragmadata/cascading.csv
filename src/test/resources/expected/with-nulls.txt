id	first name	last name
1	Jordan	Halterman
2	Tom	null
3	Harrison	Cavallero
4	null	Ahmed
5	Tristan	Reid
6	Mark	McKelvy
